src/internal/m365/onedrive/restore.go

```diff
@@ -468,7 +459,7 @@ func restoreItem(
 			itemData,
 			ctr)
 		if err != nil {
-			if errors.Is(err, graph.ErrItemAlreadyExistsConflict) && restoreCfg.OnCollision == control.Skip {
+			if errors.Is(err, graph.ErrItemAlreadyExistsConflict) && rcc.RestoreConfig.OnCollision == control.Skip {
```

Comment:
```suggestion
			if errors.Is(err, graph.ErrItemAlreadyExistsConflict) &&
			rcc.RestoreConfig.OnCollision == control.Skip {
```

Reply:
Personally, I'm not much of a fan of multilining conditional expressions unless they're breaking line length linters, so I'm going to leave these as they are.